sdk/storage/storage-queue/src/QueueServiceClient.ts

```diff
@@ -106,6 +107,35 @@ export interface ServiceListQueuesSegmentOptions {
   include?: ListQueuesIncludeType;
 }
 
+/**
+ * Options to configure Queue Service - List Queues operation
+ *
+ * @export
+ * @interface ServiceListQueuesOptions
+ */
+export interface ServiceListQueuesOptions {
+  /**
+   * Aborter instance to cancel request. It can be created with Aborter.none
+   * or Aborter.timeout(). Go to documents of {@link Aborter} for more examples
+   * about request cancellation.
+   *
+   * @type {Aborter}
+   * @memberof AppendBlobCreateOptions
```

Comment:
`AppendBlobCreateOptions` my bad from copy-n-paste.  Should be `ServiceListQueuesOptions`